### ISSUE TITLE
Improvement for screen reader users

### ISSUE THREAD

**Issue by aryanchoudharypro** [woheller69/whisperIME#12]
Hello, first off, I would like to start by saying that this app is very useful for us, but there are some issues. For example, the screen readers cannot read the element names in the UI because they are not properly labeled. Please label them according to the Android accessibility guidelines. Also, there should be some kind of feedback to let us know that the recording has been started. I would like to thank you for this project and hope that it keeps growing.
